profiling/README.md

```diff
@@ -0,0 +1,101 @@
+# Benchmark
+
+The benchmark script applies HTTP, gRPC, and TCP workloads with
+the benchmark utilities wrk2, strest-grpc, and iperf.
+The script logs to files and generates a summary CSV with the
+99th percentile latencies (or GBit/s for TCP).
+
+You can read the summary file or the individual result files for
```

Comment:
It's jarring that we jump right into how to read the result files without talking about how to generate them.  I feel like this is missing instructions on how to *run* the benchmark, profile, and memory profile.

Reply:
the included scripts are useful as a starting point, but we will often want to take benchmarks or profiles of custom workloads.  It would be nice to have instructions for running the scripts as well as instructions for gathering benchmarks or profiles on custom workloads.